packages/common/src/store/player/slice.ts

```diff
@@ -1,20 +1,18 @@
-import { UID, ID, Collectible, Nullable } from '@audius/common'
 import { createSlice, PayloadAction } from '@reduxjs/toolkit'
 
-import NativeMobileAudio from 'audio/NativeMobileAudio'
+import { UID, ID, Collectible } from '../../models'
+import { Nullable } from '../../utils'
 
-import { AudioState } from './types'
+import { Audio } from './types'
```

Comment:
just OOC, why the name change? Might make sense make the name more specific since it's exported at the top level of common

Reply:
yeah def agreed, updating it to `AudioPlayer`